changelogs/api-management-service.md

```diff
@@ -2,15 +2,21 @@
 
 ## Release - API Management service: April 2023
 
-A regular Azure API Management service update was started on . This release will continue to rollout through May 2023.
+### Highlights
+
+1. [Workspaces for Azure API Management is now in public preview.](https://azure.microsoft.com/updates/public-preview-workspaces-in-azure-api-management/)
+2. [Incoming traffic to Azure API Management's gateway can be secured to clients running in a virtual network through Azure Private Link.](https://azure.microsoft.com/updates/generally-available-azure-private-link-support-in-azure-api-management/)
+3. [Microsoft bolsters cloud-native security in Defender for Cloud with new API security capabilities](https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-bolsters-cloud-native-security-in-defender-for-cloud/ba-p/3801818)
```

Comment:
3 and 4 are the same. Could you change them to this:

`3. [Azure API Management and Microsoft Defender for APIs integration is now in preview](https://azure.microsoft.com/updates/public-preview-azure-api-management-and-microsoft-defender-for-apis-integration/). [Learn how Microsoft bolsters cloud-native security in Defender for Cloud with new API security capabilities](https://aka.ms/DfAPIlaunch).`